### DISCUSSION TITLE
Space push internal error

### DISCUSSION THREAD

**Discussion by gebardas** [deta/space-docs#96] [General]
Since this morning:

"2023/05/08 10:09:23 All micros built successfully
2023/05/08 10:09:29 build complete...
2023/05/08 10:09:29 created revision: glowworm-ajzs

i Updating your Builder instance with the new revision...

2023/05/08 10:09:35 starting update...
2023/05/08 10:09:35 fetching release configuration...
2023/05/08 10:09:35 allocating resources..
2023/05/08 10:09:35 configuring resource 1 out of 1...
2023/05/08 10:09:35 error: internal error
                                                      
! Failed to update Builder instance. Please try again!"

**Comment by oxoovo** [comment]
I have the same problem since yesterday.

**Reply by sudo-subg** [reply]
hey @gebardas, @oxoovo this should be resolved now. Please try to deploy now and let me know.

**Reply by oxoovo** [reply]
I didn't use the [run command](https://deta.space/changelog/post-13#revamped-python-micros), but now it works.
